openmcyclus/depletion.py

```diff
@@ -3,156 +3,192 @@
         csv file is assumed to be named "micro_xs.csv". This will need to
         be relative to Cyclus input file location
 
         Parameters:
         -----------
+        path: str
+            path of directory holding the files for/from OpenMC        
 
         Returns:
         --------
         microxs: object
             microscopic cross section data
         '''
-        microxs = od.MicroXS.from_csv(self.path / "micro_xs.csv")
+        microxs = od.MicroXS.from_csv(str(path + "micro_xs.csv"))
         return microxs
 
-    def run_depletion(self):
+    def run_depletion(self, path):
         '''
         Run the IndependentOperator class in OpenMC to perform
         transport-independent depletion.
 
         Parameters:
-        -----------
+        -----------        
+        path: str
+            path of directory holding the files for/from OpenMC
 
         Outputs:
         --------
         depletion_results.h5: database
             HDF5 data base with the results of the depletion simulation
         '''
-        model = self.read_model()
-        micro_xs = self.read_microxs()
+        model = self.read_model(path)
+        micro_xs = self.read_microxs(path)
         ind_op = od.IndependentOperator(model.materials, micro_xs,
-                                        str(self.path / self.chain_file))
-        ind_op.output_dir = self.path
-        integrator = od.PredictorIntegrator(ind_op, np.ones(
-            self.timesteps*30), power=self.power*1000*self.conversion_factor, 
+                                        str(path + self.chain_file))
+        ind_op.output_dir = path
+        integrator = od.PredictorIntegrator(
+            ind_op,
+            np.ones(
+                self.timesteps *
+                30),
+            power=self.power *
+            1e6*
+            self.conversion_factor,
             timestep_units='d')
         integrator.integrate()
 
         return
 
-    def create_recipe(self):
+    def get_spent_comps(self, material_ids, path):
         '''
-        Converts the depleted material compositions to an XML file readable
-        by cyclus.
+        Creates a list of each of the spent fuel compositions from the 
+        OpenMC depletion
 
         Parameters:
         -----------
+        material_id: list of strs
+            material ids for the assembly materials in the OpenMC model
+        path: str
+            path of directory holding the files for/from OpenMC
 
-        Outputs:
+        Returns:
         --------
-        {self.path}/{self.prototype}_fuel.xml : XML file
-            File containing the depleted material composition in the
-            required format for Cyclus. The path and prototype names are read
-            in from the class instantiation. Path will need to
-            be relative to Cyclus input file location
+        spent_comps: list of dicts
+            list of the compositions from the OpenMC model
```

Comment:
`dicts` isn't a python type, but `dict` is.
```suggestion
        spent_comps : list of dict
            List of the compositions from the OpenMC model.
```